windows-driver-docs-pr/stream/ksproperty-cameracontrol-extended-roi-ispcontrol.md

```diff
@@ -99,6 +99,9 @@ The table below contains the descriptions and requirements for the **KSCAMERA\_E
 </tbody>
 </table>
 
+## Adjustable Output Window Considerations
+The 3A ROI coordinates sent to the canmera are sent relative to the current output window of the camera. If the field of view has been modified due to use of a control such as [Zoom, Pan or Tilt](https://docs.microsoft.com/en-us/windows-hardware/drivers/stream/propsetid-vidcap-cameracontrol) or [Digital Window](https://docs.microsoft.com/en-us/windows-hardware/drivers/stream/digital-window-overview), the camera is responsible for mapping the provided coordinates back to the sensor's full field of view, taking the current output window into consideration to ensure that the camera's 3A algorithm is targeting the correct spot.
```

Comment:
Good point about clarifying that it doesn't have to happen in the camera. how about if I reword as follows (addition bolded):

>The 3A ROI coordinates sent to the camera are sent relative to the current output window of the camera. If the field of view has been modified due to use of a control such as [Zoom, Pan or Tilt](https://docs.microsoft.com/en-us/windows-hardware/drivers/stream/propsetid-vidcap-cameracontrol) or [Digital Window](https://docs.microsoft.com/en-us/windows-hardware/drivers/stream/digital-window-overview), the camera **(or component implementing the Digital Window/PTZ control)** is responsible for mapping the provided coordinates back to the sensor's full field of view, taking the current output window into consideration to ensure that the camera's 3A algorithm is targeting the correct spot.

Can you elaborate a bit on the second point? If the component doing the zoom is sitting after the PDMFT, what sort of remapping does it need to do? The PDMFT is operating on the raw camera feed which presumably in this case would not be impacted by the DW/PTZ window. That is unless the consuming application is applying a custom ROI (e.g,, customer zooms in with the Camera App and then does tap-to-3A. In that case, that DMFT needs to remap the coordinate in the same way that I've described so far.

Reply:
if the app want to draw face rectangles, then the coordinates attached by platformDMFT (which corresponds to what FoV PDMFT processes) may not apply correctly if a driver component closer to the app then applies a digital zoom. That component needs to crack open that metadata and adjust coordinates relative to the FoV it ouputs or even withdraw that roi if not in frame.

i.e. if we have 0) sensor-> 1) avstream -> 2) pdmft (face roi metdata) -> 3) custom DMFT applying zoom -> 4) app reads face roi metadata, because of 3) those coordinates are not valid (maybe even that roi is not in frame anymore).

Reply:
Got it, you're saying for when it's going the other way. Will update to cover.